docs/restaurant.rb

```diff
@@ -0,0 +1,16 @@
+class Restaurant
+  attr_accessor :list_dishes
+
+  def initialize
+    @list_dishes = { "pesto pasta" => 5,
+      "ravioli" => 10,
+      "pigeon" => 5,
+      "arabic herbs" => 10,
+      "mud pie" => 5 }
+  end
+
+  def menu
```

Comment:
Good method names are usually verbs, giving you an idea of the one thing the method does. I'd call this `print_menu` or similar.